docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '2'
+
+services:
+  microtransactionscommandhandler:
+    container_name: microtransactionscommandhandler
+    image: petprojects/micro-transactions-command-handler:dev
+    build:
+      context: ./src/Mts.CommandHandler
+      dockerfile: Dockerfile
+      args:
+        source: ./build
+    ports:
+      - "5000:80"
```

Comment:
We need an exposed port for this console app?

Reply:
Nope we don't need. Forgot to remove.